homeassistant/components/tplink/light.py

```diff
@@ -215,7 +214,6 @@ def __init__(
         if len(self._attr_supported_color_modes) == 1:
             # If the light supports only a single color mode, set it now
             self._fixed_color_mode = next(iter(self._attr_supported_color_modes))
-        self._async_call_update_attrs()
```

Comment:
Unnecessary as called in `async_added_to_hass`.

Reply:
This is now called by the parent's `__init__`, see [`6c971d1` (#120868)](https://github.com/home-assistant/core/pull/120868/commits/6c971d1c7c88889c35ed57e4afe7264261cbd5ae)